packages/online-playground/package.json

```diff
@@ -11,10 +11,11 @@
   "devDependencies": {
     "@vitejs/plugin-vue": "^5.2.1",
     "execa": "^9.5.1",
+    "typescript": "^5.7.2",
```

Comment:
Do we need this? ts is installed at the root to make sure only one version is used (currently 5.6 because other things break)